content/en/guides/os-commercial-ecosystem.md

```diff
@@ -0,0 +1,182 @@
+---
+title: Creating an Open Source Commercial Ecosystem
+---
+
+An open source project is successful when it has a thriving ecosystem of companies and products built around it.  So what exactly is an open source commercial ecosystem and how can organizations help create and support sustainable open source projects? There are concrete ways to create confidence in a project’s long-term viability, encourage companies to create commercial products and services on top of open source projects, and re-invest in them with contributions, content, and other resources. These practices for enabling ecosystem development help set your projects up for success.
```

Comment:
The implication here is that a commercial ecosystem is _the_ way for a project to be successful. That's not the case.

Suggested change: "One way for an open source project to be successful is to have a thriving…"